app/routes/events/components/EventAdministrate/RegistrationTables.tsx

```diff
@@ -65,31 +66,60 @@ const getPoolName = (pools, poolId) => {
   return pool && pool.name;
 };
 
+type RegistrationPillProps = {
+  state: string;
+  reason: string;
```

Comment:
I would prefer if these were a bit more self-explanatory. Especially state is confusing in the react world

Reply:
Do you have an example? I find `reason` to be good, but I can agree `state` might not be the best. At the same time, the pills do have an x amount of states, so it sort of makes sense. But ye, in the react world, might not be the best.

Reply:
I.e. `registrationState` is a better name here.

Reply:
@LudvigHz What about `status`? I feel like `registrationInfo.registrationState` is a bit much.

Reply:
Status is better :+1: